core/__tests__/classes/loggedModel.ts

```diff
@@ -48,6 +49,7 @@ describe("classes/loggedModel", () => {
         "foo:stuff",
         "foo;stuff",
         "foo$stuff",
+        "foo-is-long-really-long-really-really-long-really-really-really-long-really-really-really-really-long-really-really-really-really-really-long-really-really-really-long-like-192-characters-long",
```

Comment:
🤣

Reply:
might I suggest testing `"a".repeat(191)` and `"a".repeat(192)`

Reply:
That's no fun.